mappings/net/minecraft/entity/EntityType.mapping

```diff
@@ -46,6 +47,13 @@ CLASS net/minecraft/class_1299 net/minecraft/entity/EntityType
 	METHOD method_20210 isIn (Lnet/minecraft/class_3494;)Z
 		ARG 1 tag
 	METHOD method_20814 isSpawnableFarFromPlayer ()Z
+	METHOD method_29496 isInvalidSpawn (Lnet/minecraft/class_2680;)Z
+		COMMENT Returns whether the EntityType can spawn inside the given block.
+		COMMENT
+		COMMENT Non fire-immune mobs can't spawn in / on blocks dealing fire damage.
+		COMMENT Mobs can't spawn in wither roses, sweet berry bush, or cacti.
+		COMMENT
+		COMMENT This can be overwritten via @link EntityType.Builder.allowSpawningInside
```

Comment:
```suggestion
		COMMENT Returns whether the EntityType can spawn inside the given block.
		COMMENT
		COMMENT <p>By default, non-fire-immune mobs can't spawn in/on blocks dealing fire damage.
		COMMENT Any mob can't spawn in wither roses, sweet berry bush, or cacti.
		COMMENT
		COMMENT <p>This can be overwritten via {@link EntityType.Builder#allowSpawningInside(Block[])}
```